dockerfiles/init/modules/codenvy/templates/machine.properties.erb

```diff
@@ -86,7 +86,7 @@ che.docker.network_driver=<%= scope.lookupvar('codenvy::che_machine_docker_netwo
+che.workspace.java.options=<%= scope.lookupvar('codenvy::workspace_java_options') %> <% if ! @http_proxy_for_codenvy_workspaces.empty? or ! @https_proxy_for_codenvy_workspaces.empty? -%>-Dhttp.proxySet=true<% end -%><% if ! @http_proxy_for_codenvy_workspaces.empty? -%><% if ! @http_proxy_for_codenvy_workspaces.empty? and @http_proxy_for_codenvy_workspaces.include? '@' -%> -Dhttp.proxyUser=<%= @http_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split('@')[0].split(':')[0] %> -Dhttp.proxyPassword=<%= @http_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split('@')[0].split(':')[1] %> -Dhttp.proxyHost=<%= @http_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split('@')[1].split(':')[0] %> -Dhttp.proxyPort=<%= @http_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split('@')[1].split(':')[1].gsub(/\/.*/,'') %><% else -%> -Dhttp.proxyHost=<%= @http_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split(':')[0] %> -Dhttp.proxyPort=<%= @http_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split(':')[1].gsub(/\/.*/,'') %><% end -%><% end -%><% if ! @https_proxy_for_codenvy_workspaces.empty? -%><% if @https_proxy_for_codenvy_workspaces.include? '@' -%> -Dhttps.proxyUser=<%= @https_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split('@')[0].split(':')[0] %> -Dhttps.proxyPassword=<%= @https_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split('@')[0].split(':')[1] %> -Dhttps.proxyHost=<%= @https_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split('@')[1].split(':')[0] %> -Dhttps.proxyPort=<%= @https_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split('@')[1].split(':')[1].gsub(/\/.*/,'') %><% else -%> -Dhttps.proxyHost=<%= @https_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split(':')[0] %> -Dhttps.proxyPort=<%= @https_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split(':')[1].gsub(/\/.*/,'') %><% end -%><% end -%><% if ! @no_proxy_for_codenvy_workspaces.empty? -%> -Dhttp.nonProxyHosts='<%= @no_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').gsub(',','|').split("|").uniq.join("|") %>|'<% end -%>
```

Comment:
Looks like this statement can be simplified: 
`.split(",").uniq.join("|") %>|'<% end -%>`